packages/common/src/extendedObject3D.ts

```diff
@@ -128,6 +130,32 @@ export class ExtendedObject3D extends Object3D {
   public setAction(name: string) {
     console.warn(`[enable3d] setAction(${name}) is deprecated. Use animation.play(${name}) instead!`)
   }
+
+  public traverse(callback:( object: ExtendedObject3D ) => any): void {
+    callback(this)
+    const children = this.children
+    for ( let i = 0, l = children.length; i < l; i ++ ) {
+      children[ i ].traverse( callback )
+    }
+  }
```

Comment:
I prefere:

```suggestion
public traverse(callback: (object: ExtendedObject3D) => any): void {
  super.traverse(callback as any)
}
```

Reply:
Yeah, in this way is 1000 time better! Thanks!